beets/ui/commands.py

```diff
@@ -1210,6 +1213,12 @@ def update_items(lib, query, album, move, pretend, fields):
             fields.append('path')
         items, _ = _do_query(lib, query, album)
 
+        item_fields = fields or library.Item._media_fields
+        album_fields = fields or library.Album._fields.keys()
```

Comment:
There's nothing specific to do here, but I just wanted to highlight that this looks somewhat odd when put next to each other (`Item` uses `_media_fields`, `Album` uses all `_fields`). There's a good reason for this (the former needs to pick only the relevant fields that might come from files); it just looks somewhat odd.

Reply:
Setting `item_fields` to `_fields.keys()` does fix the the failing tests, but I'm investigating other solutions in case there's a tighter fix.

Reply:
yup, found a smaller set of fields to update. It's just `_media_fields` and conditionally `path`